app/posts/apply-for-teacher-training/2020-01-16-question-protocol-for-pilot.md

```diff
@@ -0,0 +1,1722 @@
+    },
+    why: [{
+      who: ["candidate"],
+      what: "needs to provide evidence that their international qualification demonstrates a similar level and breadth as the equivalent UK qualification"
+    }, {
+      who: ["candidate"],
+      what: "may already have a statement of comparability from NARIC"
+    }, {
+      who: ["provider"],
+      what: "needs to look for evidence that a qualification is of a standard equivalent to GCSE grade 4, not only in terms of its level, but also in terms of its breadth. Details of qualification can help determine if equivalent to GCSE"
+    }],
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    accuracy: ["Providers will ask to see hard copies for any qualifications awarded"],
+    notes: ["Candidates can request a statement of comparability from NARIC, or speak to a TTA who can certify equivalence"]
+  }
+}) }}
+
+## Science GCSE or equivalent
+
+Only required if applying for a Primary course (i.e. intend to train to teach pupils aged 3-11).
+
+Candidates can update their answers before (and 5 working days after) submitting their application.
+
+{{ appProtocol({
+  title: "Qualification - type",
+  items: {
+    question: "Type of qualification",
+    required: "Yes (if applying for a Primary course)",
+    api: {
+      param: "qualifications.gcses[].qualification_type",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#qualification-object"
+    },
+    why: [{
+      who: ["provider"],
+      what: "needs to ensure candidates have achieved a standard equivalent to a grade 4 in a GCSE science subject"
+    }],
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    accuracy: [
+      "Providers will ask to see hard copies for any qualifications awarded",
+      "In the future we can check any post-14 qualifications (awarded after 2006 in England, Wales and Northern Ireland) against the Learning Records Service"]
+  }
+}) }}
+
+{{ appProtocol({
+  title: "Qualification - grade",
+  items: {
+    question: "What grade is your English qualification?",
+    required: "Yes (if applying for a Primary course)",
+    api: {
+      param: "qualifications.gcses[].grade",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#qualification-object"
+    },
+    why: [{
+      who: ["provider"],
+      what: "needs to ensure candidates have achieved a standard equivalent to a grade 4 in a GCSE science subject"
+    }],
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    accuracy: [
+      "Providers will ask to see hard copies for any qualifications awarded",
+      "In the future we can check any post-14 qualifications (awarded after 2006 in England, Wales and Northern Ireland) against the Learning Records Service"],
+    notes: ["Do candidates need to specify which subjects they obtained each grade in? (e.g. double or triple award science)?"]
```

Comment:
ITT criteria says:
> a standard equivalent to a grade 4 in the GCSE examination _in a science subject_

Which I read as "any".

But, HEI providers are asking to know which subjects.

Reply:
Updated note.